.github/workflows/tests.yml

```diff
@@ -12,7 +12,7 @@ on:
 
 jobs:
   unit-tests:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-20.04
```

Comment:
Needed since https://github.com/actions/setup-python/issues/544

We could drop `3.6` since it's EOL but that's for you to decide :)